src/utils/constants.ts

```diff
@@ -136,3 +136,8 @@ export const MAX_LEGS_LENGTH = 10;
  * Default CDD ID associated with an Identity on chain. Used for Identities onboarded without PUIS
  */
 export const DEFAULT_CDD_ID = '0x0000000000000000000000000000000000000000000000000000000000000000';
+
+/**
+ * Maximum lag of block processing allowed in the middleware to indicate that the middleware is in a synced state
+ */
+export const MAX_MIDDLEWARE_LAG = new BigNumber(5);
```

Comment:
Maybe 10? IDK, but I think ~1 minute is reasonable.

30 seconds seems like it could happen on big batch blocks. IDK the right value though